spel/scripts/amigen7-build.sh

```diff
@@ -9,10 +9,12 @@ AMIGENBOOTSIZE="${SPEL_AMIGENBOOTSIZE:-500m}"
 AMIGENBRANCH="${SPEL_AMIGENBRANCH:-master}"
 AMIGENBUILDDEV="${SPEL_AMIGENBUILDDEV:-/dev/nvme0n1}"
 AMIGENCHROOT="${SPEL_AMIGENCHROOT:-/mnt/ec2-root}"
+AMIGENFSTYPE="${SPEL_AMIGENFSTYPE:-ext4}"
 AMIGENMANFST="${SPEL_AMIGENMANFST}"
 AMIGENPKGGRP="${SPEL_AMIGENPKGGRP:-core}"
 AMIGENREPOS="${SPEL_AMIGENREPOS}"
 AMIGENREPOSRC="${SPEL_AMIGENREPOSRC}"
+AMIGENROOTNM="${SPEL_AMIGEN8ROOTNM:-UNDEF}"
```

Comment:
If the var is going to be used for both el7 and el8 builds, then the var name should not include the major version, i.e. `spel_amigenrootnm` and `SPEL_AMIGENROOTNM`

Reply:
Otherwise, we need to add a _separate_ var. i.e. `spel_amigen7rootnm` and `SPEL_AMIGEN7ROOTNM`

Reply:
Also, these envs need to be pass into the el7 builders...